### ISSUE TITLE
'KeyError: Texture' when importing VRMat file

### ISSUE THREAD

**Issue by RobBollons** [bdancer/blender-for-vray#175]
Importing a vrmat file that has been exported via 3DS Max results in a 'KeyError' for 'texture'.

I've attached an example of a file that has the error.

Blender (With V-Ray Additions) OSX v2.77.3
Exporter v3.2116

`File "/Applications/ChaosGroup/Blender 2.77 (With V-Ray Additions)/blender.app/Contents/Resources/2.77/scripts/addons/vb30/nodes/importing.py", line 367, in CollapseToValue
    texture = pluginAttrs['texture']
KeyError: 'texture'`

[1___Atique_Pewter.vrmat.zip](https://github.com/bdancer/blender-for-vray/files/421019/1___Atique_Pewter.vrmat.zip)

**Comment by bdancer** [comment]
Does it work with the VRmat node?

**Comment by JuhaW** [comment]
Yes, works with VRmat node but when import same error than in the first post.

**Comment by RobBollons** [comment]
I concur with @JuhaW. The VRMat node works nicely but its the import that fails.
